src/main/java/org/beanpod/switchboard/dto/mapper/LogMapper.java

```diff
@@ -11,4 +14,17 @@
   List<LogModel> toLogModels(List<LogEntity> logEntityList);
 
   LogModel toLogModel(LogEntity logEntity);
+
+  LogEntity toLogEntity(LogDto logDto);
+
+  LogDto toLogDto(CreateLogRequest createLogRequest);
+
+  LogDto toLogDto1(LogEntity logEntity);
+
+  /* to convert string to OffsetDateTime
+   * value must be of YYYY-MM-DDTHH:mm:ss+00:00
+   */
+  default OffsetDateTime map(String value) {
+    return OffsetDateTime.parse(value);
+  }
```

Comment:
This would fit well in `util/DateUtil.java`

Reply:
I don't think it's appropriate to be in DateUtil class since the class needs to be mapper (@Mapper). I found two people so far did the same way I did: https://stackoverflow.com/questions/59616631/mapping-2-string-fields-to-offsetdatetime-with-mapstruct

An alternative would be extracting this method to a new class, but I don't see why would we do that.

Reply:
I see, I missed that the first time I read.